docs/source/quick_start.rst

```diff
@@ -0,0 +1,160 @@
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users (and beginners, in particular).  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+Download Latest and Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" should be stable from one release to the next, but check periodically to be sure 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/latest>`
+
+.. code-block:: sh
+  
+  # Uncomment according to the type of file you've downloaded (zip or tar)
+  unzip kokkos-x.y.z.zip 
+  # tar -xzf kokkos-x.y.z.tar.gz
+  cd kokkos-x.y.z
+  mkdir build
+  cd build
+
+
+Basic Configuration
+-------------------
+
+.. note::
+
+  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
+
+
+
+OpenMP (CPU Parallelism)
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: sh
+
+  cmake \
+   -DKokkos_ENABLE_OPENMP=ON \
+   -DCMAKE_BUILD_TYPE=Release ..
+
+
+.. note::
+
+  Kokkos will attempt to autodetect GPU microarchitecture, but it is also possible to specify the desired `GPU architecture <https://kokkos.org/kokkos-core-wiki/keywords.html#gpu-architectures>`_.  In scenarios where a backend (e.g., CUDA, HIP) is enabled, Kokkos will execute serially on the "host" (CPU).  
+
+CUDA (CPU and GPU)
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: sh
+
+  cmake \
+    -DKokkos_ENABLE_CUDA=ON \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+
+HIP (CPU and GPU)
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: sh
+
+  cmake \
+    -DKokkos_ENABLE_HIP=ON \
+    -DCMAKE_CXX_COMPILER=hipcc \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+
+Basic Configuration Examples for Spack Builds
+----------------------------------------------
```

Comment:
It might be obvious to you but some might not realize that this is meant as an alternative to the previous section.
Also you could just refer to the spack section in the build reference as an alternative solution to install Kokkos and leave it there